### DISCUSSION TITLE
[Help Wanted] open standard notes (appimage)

### DISCUSSION THREAD

**Discussion by tuxedotje** [standardnotes/forum#2801] [General]
**Describe the issue**
A clear and concise description of what the issue is.

**To Reproduce**
find SN 

**Expected behavior**
SN should pop up in a start screen. I should be able to open SH 
But have to install SN again with run appimage 

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. iOS]  Fedora 36
 - Browser [e.g. chrome, safari]  Firefox
 - Version [e.g. 22]  3.20

**Smartphone (please complete the following information):**

**Additional context**

**Comment by JadoJodo** [comment]
I'm not involved with this project, but reading through your report here, I'm having trouble grepping the issue. For clarity, this is what I understand you're reporting:

1. You're running the StandardNotes AppImage (v3.20) on Fedora. 
2. You expect to see the StandardNotes icon in the Start Screen.
3. You do not see the StandardNotes icon in the Start Screen.
4. Re-running the AppImage file causes the StandandNotes icon to appear in the Start Screen.
5. Restarting your session/system causes the StandardNotes icon to disappear from the Start Screen.

Assuming this is correct, I would suggest the following additional details be added:

- What DE are you using?
- Which architecture AppImage are you using? ARM? x86_64? i386? 
- Does this behavior occur for other AppImages? e.g., LibreOffice
- Have you tried an older (or newer -- v3.20.2 is out) StandardNotes AppImage?
